utils/load_utils.py

```diff
@@ -0,0 +1,377 @@
+            nan_col_to_fill="Primary Cell Line Name",
+            missing_tissue_dict=CCLE_MISSING_TISSUE_NAMES,
+        )
+
+    if only_get_cells:
+        cell_line_cols = [
+            "CCLE Cell Line Name",
+            "Primary Cell Line Name",
+            "cell_line_clean",
+            "tissue",
+        ]
+        ccle_df = (
+            ccle_df.loc[:, cell_line_cols].drop_duplicates().reset_index(drop=True)
+        )
+
+    return ccle_df
+
+
+def load_depmap(
+    top_dir,
+    data_dir="depmap/data",
+    load_cell_info=False,
+    load_gene_info=False,
+    only_get_cells=False,
+):
+    """
+    Load Cancer Dependency Map viability data
+
+    Arguments:
+    ----------
+    top_dir : str
+        Where the repository top level directory is in relation to your current wd
+    data_dir : str, default "depmap/data"
+        The intermediate level folder where data are stored
+    load_cell_info : boolean, default False
+        Whether or not to load cell line map
+    load_gene_info : boolean, default False
+        Whether or not to load gene map
+    only_get_cells : boolean, default False
+        Decision to load only columns (without duplicates) of cell line info
+
+    Returns:
+    --------
+        The DepMap viability information per cell line : list of pandas.DataFrame
+    """
+    # Certain conditions must be met for only_get_cells
+    if only_get_cells:
+        assert load_cell_info, "load_cell_info must be True if only_get_cells is True"
+        assert (
+            not load_gene_info
+        ), "load_gene_info must be False if only_get_cells is True"
+
+    # Create path to DepMap data
+    depmap_dir = pathlib.Path(top_dir, data_dir)
+    depmap_filename = FILENAMES["depmap"]["viability"]
+
+    depmap_file = pathlib.Path(depmap_dir, depmap_filename)
+
+    # Load data
+    depmap_return_packet = []
+    depmap_return_packet.append(pd.read_csv(depmap_file))
+
+    if load_cell_info:
+        depmap_cell_filename = FILENAMES["depmap"]["celllines"]
+        depmap_cell_file = pathlib.Path(depmap_dir, depmap_cell_filename)
+
+        depmap_cell_df = pd.read_csv(depmap_cell_file)
+
+        depmap_cell_df = process_tissue_ccle_column(
+            input_df=depmap_cell_df,
+            recode_col_name="CCLE_Name",
+            nan_col_to_fill="cell_line_name",
+        )
+
+        depmap_cell_df = process_missing_tissues(
+            input_df=depmap_cell_df,
+            recode_col_name="cell_line_name",
+            missing_tissue_dict=DEPMAP_MISSING_TISSUE_NAMES,
+        )
+
+        depmap_return_packet.append(depmap_cell_df)
+
+    if load_gene_info:
+        depmap_gene_filename = FILENAMES["depmap"]["genes"]
+        depmap_gene_file = pathlib.Path(depmap_dir, depmap_gene_filename)
+
+        depmap_return_packet.append(pd.read_csv(depmap_gene_file, sep="\t"))
+
+    if len(depmap_return_packet) == 1:
+        depmap_return_packet = depmap_return_packet[0]
```

Comment:
Consider providing comments throughout this section to help describe why these steps are performed.

Reply:
will do!